pkg/plugins/platform/local.go

```diff
@@ -291,6 +288,52 @@ func (l *LocalApp) SiteStatus() string {
 	return siteStatus
 }
 
+// Import performs an import from a local
+func (l *LocalApp) Import() error {
+	provider, err := l.AppConfig.GetProvider()
+	if err != nil {
+		return err
+	}
+
+	err = provider.Validate()
+	if err != nil {
+		return err
+	}
+
+	if l.SiteStatus() != SiteRunning {
+		util.Warning("Site is not currently running. Starting site before performing import.")
```

Comment:
Might be just me, but this seemed weird as a warning. I would be ok with plain text personally.